images/ubuntu/Ubuntu2404-Readme.md

```diff
@@ -151,7 +150,7 @@ Both Xdebug and PCOV extensions are installed, but only Xdebug is enabled.
 | SELENIUM_JAR_PATH | /usr/share/java/selenium-server.jar   |
 
 ### .NET Tools
-- .NET Core SDK: 8.0.300
+- .NET Core SDK: 8.0.105
```

Comment:
I guess that is what caused incompatibility with the MS repo, lets ignore that for now